css/main.css

```diff
@@ -325,3 +326,112 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* Contact Form */
+.form-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5rem 0 0;
+}
+
+.cols {
+  display: flex;
+  flex-direction: column;
+}
+
+.cols > p {
+  font-family: 'Inter', sans-serif;
+  line-height: 2.75rem;
+  text-align: center;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 2rem;
+  margin-bottom: 4.375rem;
+  padding: 1.5rem;
+}
+
+.form-content {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  padding: 0 1.5rem;
+  align-self: center;
+}
+
+input {
+  border: 0;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 5%;
+  margin-bottom: 1.25rem;
+  font-family: 'Inter', sans-serif;
+  font-size: 0.9375rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+textarea {
+  height: 7.125rem;
+  margin-bottom: 1.5rem;
+  border: 0;
+  padding: 0.75rem;
+  background: #fbf8f7;
+  font-family: 'Inter', sans-serif;
+  font-size: 0.9375rem;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.5rem;
+  letter-spacing: 0;
+  text-align: left;
+  resize: vertical;
+}
+
+.form-button {
+  background-color: #7b5f4b;
+  color: #fff;
+  cursor: pointer;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 1.0625rem;
+  line-height: 1.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 3rem;
+  border: 0;
+  margin-left: 0;
+  width: 9.375rem;
+  align-self: center;
+}
```

Comment:
- [ ] The `background-color` of the button on your application is different from that of the Figma design you chose. In Figma, the `background-color` is `#F55800` while you used `#7b5f4b`. Kindly change it to match that of the Figma design. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/203837756-7e47e8cc-d6fe-4033-9e5e-b65e150ea62d.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/203837845-e48ee049-6748-4790-871b-6be198e702f5.png)